FindRange.java

```diff
@@ -1,16 +1,65 @@
-/*
- * File: FindRange.java
- * Name: 
- * Section Leader: 
- * --------------------
- * This file is the starter file for the FindRange problem.
- */
-
-import acm.program.*;
-
-public class FindRange extends ConsoleProgram {
-	public void run() {
-		/* You fill this in */
-	}
-}
-
+/*

```

Comment:
Program doesn't work accurately. It fails the following two cases described in the assignment:
```
If the user enters only one value before the sentinel, the program should report
that value as both the largest and smallest
```
So for example if someone enters 5 followed by 0 the program should report:
smallest: 5
largest: 5
Your program instead reports
smallest: 0
largest: 5
 
 
Also
```
If the user enters the sentinel on the very first input line, then no values have been
entered, and your program should display a message to that effect.
```
So if for example the user enters 0 the program should report:
No values has been entered.
Your program instead reports:
Smallest: 0
Largest: 0